docs/proposals/multi-node/restoration.md

```diff
@@ -0,0 +1,37 @@
+# Restoration of whole cluster from the backup bucket
+A cluster may need to be restarted from backup bucket if for some reason all the persistent volumes are deleted or corrupted. This situation may occur more prominently while the cluster is hibernating during maintenance and the persistent volumes are deleted.
+
+A backup bucket contains multiple full snapshots and delta snapshots.
+
+Full snapshot is the snapshot of database memory at a particular time. Restoration of ETCD cluster from full snapshot means the ETCD cluster will be bootstrapped with that database memory.
+
+Delta snapshot is the ETCD events listed together in a file. Restoring from delta snapshot means applying the ETCD events one by one onto the ETCD cluster. This takes longer than restoring from full snapshot.
+
+For restoration, ETCD Backup Restore container checks first whether the PV is avialable or not. If PV is available, ETCD BR simply starts the respective ETCD nodes. ETCD nodes connect with each other to form a cluster.
+
+If PV is not available, ETCD  BR container connects to the cloud storage and fetches the latest full snapshot and subsequent delta snapshots.
+
+In the particular scenario that we are considering, all the ETCD BR discovers that the respective PVs are unavialble. So, all of them connect to the backup bucket and fetches latest full snapshot and delta snapshots.
```

Comment:
Can you be more precise here: How do all pods determine that the data dir is gone, how is this coordinated between the pods?